### PR TITLE
Add QMPlay2

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/zaps166/QMPlay2/issues/551
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build com.github.zaps166.QMPlay2

File: com.github.zaps166.QMPlay2.json [hunk 1]
```diff
@@ -0,0 +1,150 @@
+{
+  "app-id": "com.github.zaps166.QMPlay2",
```
Comment (hfiguiere) [inline com.github.zaps166.QMPlay2.json:2]:
```suggestion
  "app-id": "io.github.zaps166.QMPlay2",
```

**Comment by Mailaender** [comment]
bot, build io.github.zaps166.QMPlay2

File: com.github.zaps166.QMPlay2.json (hunk 1 above)
Comment (hfiguiere) [reply com.github.zaps166.QMPlay2.json]: wrong assumption. It has always been the case it's now strictly enforced.

See:
https://docs.flatpak.org/en/latest/conventions.html#application-ids

File: io.github.zaps166.QMPlay2.json [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://downloads.sourceforge.net/sourceforge/libcddb/libcddb-1.3.2.tar.bz2",
+          "sha256": "35ce0ee1741ea38def304ddfe84a958901413aa829698357f0bee5bb8f0a223b"
+        },
+        {
+          "type": "script",
+          "commands": [
+            "autoreconf -fiv"
```
Comment (hfiguiere) [inline io.github.zaps166.QMPlay2.json:83]: you should also copy the config.sub and config.guess to pick the newer ones.

**Pull request closed without merging**